kbuild/v2/kpub-astore

```diff
@@ -78,12 +86,25 @@ OPTIONS:
 
 		The default is "$DEFAULT_BUILD_ROOT".
 
+    -e enkit binary to use
```

Comment:
Any particular reason why a custom enkit might be required to push these artifacts?